<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="settingsTabs">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="main_tab_4">
      <attribute name="title">
       <string>Main Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,1,1,1">
       <item>
        <widget class="QTabWidget" name="sweepTypeTabs_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>300</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="linear_tab_4">
          <attribute name="title">
           <string>Linear</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QGridLayout" name="gridLayout_linear_4">
             <item row="0" column="0">
              <widget class="QLabel" name="start_label_4">
               <property name="text">
                <string>Start : </string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="comboBox_4">
               <item>
                <property name="text">
                 <string>201</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>401</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>801</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1601</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="MHz_label_7">
               <property name="text">
                <string>MHz</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="stop_label_4">
               <property name="text">
                <string>Stop :</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="points_label_4">
               <property name="text">
                <string>Points : </string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_start_4"/>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="MHz_label_8">
               <property name="text">
                <string>MHz</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_stop_4"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="list_tab_4">
          <attribute name="title">
           <string>List</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QGridLayout" name="gridLayout_list_5">
             <item row="0" column="0">
              <widget class="QLabel" name="list_label_5">
               <property name="text">
                <string>Frequencies : </string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_list_5"/>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="list_MHz_label_5">
               <property name="text">
                <string>MHz</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="global_settings_frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QGridLayout" name="gridLayout_global_settings_7">
            <item row="3" column="1">
             <widget class="QRadioButton" name="cont_radioButton_7">
              <property name="text">
               <string>continous</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_3</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="Averaging_comboBox_7">
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>16</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="res_doubleSpinBox_7">
              <property name="maximum">
               <double>5.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="Impedance_label_7">
              <property name="text">
               <string>Impedance : </string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="Impedance_radioButton_n_7">
              <property name="text">
               <string>No</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="res_label_7">
              <property name="text">
               <string>Resolution : </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="Impedance_radioButton_y_7">
              <property name="text">
               <string>Yes</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QRadioButton" name="Calibration_radioButton_y_7">
              <property name="text">
               <string>Yes </string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_2</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QRadioButton" name="Calibration_radioButton_n_7">
              <property name="text">
               <string>No</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_2</string>
              </attribute>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="posMov_label_7">
              <property name="text">
               <string>Positioner Movment : </string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="Averaging_label_7">
              <property name="text">
               <string>Averaging : </string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QRadioButton" name="discrete_radioButton_7">
              <property name="text">
               <string>discrete</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_3</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="Calibration_label_7">
              <property name="text">
               <string>Calibration : </string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="res_label_degrees_7">
              <property name="text">
               <string>degrees</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="hardware_settings_gridLayout_6">
         <item row="1" column="1">
          <widget class="QLineEdit" name="pos_alias_lineEdit_6"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="BaudRate_label_6">
           <property name="text">
            <string>Baud Rate :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="pos_Alias_label_6">
           <property name="text">
            <string>Positioner Alias : </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="GPIB_addr_comboBox_6">
           <property name="currentIndex">
            <number>16</number>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>13</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>17</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>18</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>21</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>22</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>23</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>26</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>27</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>28</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>29</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="GPIB_addr_label_6">
           <property name="text">
            <string>GPIB Address : </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="BaudRate_comboBox_6">
           <item>
            <property name="text">
             <string>1200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14400</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="settings_bttns_frame_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QDialogButtonBox" name="buttonBox">
              <property name="standardButtons">
               <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="positioner_tab_4">
      <attribute name="title">
       <string>Positioner Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QFrame" name="arrows_frame_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QToolButton" name="up_toolButton_4">
            <property name="text">
             <string>...</string>
            </property>
            <property name="shortcut">
             <string>W</string>
            </property>
            <property name="arrowType">
             <enum>Qt::UpArrow</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QToolButton" name="left_toolButton_4">
            <property name="text">
             <string>...</string>
            </property>
            <property name="shortcut">
             <string>A</string>
            </property>
            <property name="arrowType">
             <enum>Qt::LeftArrow</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="right_toolButton_4">
            <property name="text">
             <string>...</string>
            </property>
            <property name="shortcut">
             <string>D</string>
            </property>
            <property name="arrowType">
             <enum>Qt::RightArrow</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QToolButton" name="down_toolButton_4">
            <property name="text">
             <string>...</string>
            </property>
            <property name="shortcut">
             <string>S</string>
            </property>
            <property name="arrowType">
             <enum>Qt::DownArrow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="pos_display_layout_4">
         <item row="0" column="0">
          <widget class="QFrame" name="pan_frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <widget class="QLabel" name="pan_label_4">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>20</y>
              <width>31</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Pan</string>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLCDNumber" name="tilt_lcdNumber_4">
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Filled</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QFrame" name="tilt_frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <widget class="QLabel" name="tilt_label_4">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>20</y>
              <width>21</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Tilt</string>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLCDNumber" name="pan_lcdNumber_4">
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="offsetBttns_layout_4">
         <property name="spacing">
          <number>25</number>
         </property>
         <item>
          <widget class="QPushButton" name="getCurrentOffsets_Button_4">
           <property name="text">
            <string>Get Current Offsets</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="zeroOffsets_Button_4">
           <property name="text">
            <string>Zero Offsets</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="aligntoCenterButton_4">
           <property name="text">
            <string>Align to Center</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="AngleOffdoubleSpinBox_2">
             <property name="minimum">
              <double>-90.000000000000000</double>
             </property>
             <property name="maximum">
              <double>90.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Set Angle Offset: </string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Ok</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>593</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
 </buttongroups>
</ui>
